Apps/Sandcastle/gallery/Clock.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Configure the Viewer clock.">
+    <meta name="cesium-sandcastle-labels" content="Beginner">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+// Create a clock that loops on Christmas day 2013 and runs in 4000x real time.
+var clock = new Cesium.Clock({
+   startTime : Cesium.JulianDate.fromIso8601("2013-12-25"),
+   currentTime : Cesium.JulianDate.fromIso8601("2013-12-25"),
+   stopTime : Cesium.JulianDate.fromIso8601("2013-12-26"),
+   clockRange : Cesium.ClockRange.LOOP_STOP, // loop when we hit the end time
+   clockStep : Cesium.ClockStep.SYSTEM_CLOCK_MULTIPLIER,
+   multiplier : 4000 // how much time to advance each tick
+});
+
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    clockViewModel : new Cesium.ClockViewModel(clock)
+});
+
+viewer.scene.globe.enableLighting = true;
+
+Sandcastle.addToolbarButton('Reset Current Time', function() {
+    var resetTime = viewer.clockViewModel.startTime;
+    viewer.clockViewModel.currentTime = resetTime;
+    viewer.timeline.updateFromClock();
+});
+
+Sandcastle.addToolbarButton('Slow Down Clock', function() {
+    if (viewer.clockViewModel.multiplier > 0) {
```

Comment:
Can you just change this to divide the value by  2 (and change the speed up to multiply by 2)?  It was hard to notice the slowdown/speedup since it starts at 4000 compared to 3900.  I would also remove these if checks because they will no longer be needed.